src/services/Cities.ts

```diff
@@ -0,0 +1,17 @@
+import { Worker } from "../moudels/Worker";
```

Comment:
move to city module files (there is no new types that is very different from city but a function related to city).